library/core/src/test/java/com/google/android/exoplayer2/trackselection/DefaultTrackSelectorTest.java

```diff
@@ -1078,6 +1078,33 @@ public void testSelectTracksWithinCapabilitiesAndForceLowestBitrateSelectLowerBi
     assertThat(result.selections.get(0).getSelectedFormat()).isEqualTo(lowerBitrateFormat);
   }
 
+  /**
+   * Tests that track selector will select audio tracks with higher bitrate when {@link Parameters}
+   * indicate highest bitrate preference, even when tracks are within capabilities.
```

Comment:
The "even when tracks are within capabilities" part doesn't make much sense here. Maybe say:
"Tests that track selector will select the highest supported audio track when {@link Parameters#forceHighestSupportedBitrate} is set."

Reply:
Yes, the other comment can be improved too. 

Aside: We generally don't provide javadocs for tests as long as the test name itself is descriptive enough. This class seems to be an exception to that rule :)